rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -2899,10 +1047,27 @@ def test_db_newer_than_software_raises_error(data_dir, username, sql_vm_instruct
 
 def test_upgrades_list_is_sane():
     idx = None
-    for idx, entry in enumerate(UPGRADES_LIST):
+    for list_index, entry in enumerate(UPGRADES_LIST):
+        # When support for older versions was dropped we removed them from the list of assets.
+        # Adjust the test to account for this shift.
+        idx = list_index + 25
```

Comment:
you can do this smarter. Where `25` is a constant you defined somewhere above, I don't remember where or exact constant value. But you understand what I mean.
```suggestion
    for idx, entry in enumerate(UPGRADES_LIST, start=MIN_USER_DB_SUPPORTED_VERSION):
```